core/src/main/java/com/netflix/conductor/core/config/ConductorCoreConfiguration.java

```diff
@@ -87,4 +91,12 @@ public ExecutorService executorService(ConductorProperties conductorProperties)
         return eventQueueProviders.stream()
             .collect(Collectors.toMap(EventQueueProvider::getQueueType, identity()));
     }
+
+    @Bean
+    public Map<String, Evaluator> getEvaluators() {
+        return new HashMap<>() {{
+            put(ValueParamEvaluator.NAME, new ValueParamEvaluator());
```

Comment:
This can follow a similar pattern as the ones used above for `getTaskMappers` and `getEventQueueProviders` wherein we rely on Spring to collect all Evaluator implementations and make it available in the map. This would make it easier to add any new evaluator implementations in the future.

Reply:
Working on this...

Reply:
Fixed this.